source/features/blocks/api/transformers.ts

```diff
@@ -9,6 +9,7 @@ export const blockInfoTransformer = (b: BlockInfoFragment): IBlockInfo => ({
   id: b.id ? b.id : '',
   number: b.number ? b.number : 0,
   size: b.size,
+  slotNo: b.slotNo ? b.slotNo : 0,
```

Comment:
It's actually invalid to be assigning 0 as there are some blocks not created in slots: The genesis and Epoch Boundary Blocks (EBBs). Since this is out of the PR's scope I'll make an issue to address this